### ISSUE TITLE
The readme claims wrong version is installable through hex

### ISSUE THREAD

**Issue by mradke** [mailman-elixir/mailman#78]
In the readme it says we need to specify the deps as follows:

```
{:mailman, "~> 0.4.0"},
{:eiconv, github: "zotonic/eiconv"}
```

This does not work since `No matching version for mailman ~> 0.4.0 (from: mix.exs) in registry` which is correct according to https://hex.pm/packages/mailman

What is the currently recommended approach to use this library? Stay with the `0.3.0` release or install `0.4.0` from Github?

**Comment by skosch** [comment]
True! Thanks for catching that @mradke. I think @kamilc needs to upload the new version to hex.pm? 

Until then you can use `github: "kamilc/mailman"` instead – it's what I do :)

**Comment by mradke** [comment]
Thanks for reaching out. I'm also using it that way, just wanted some clarification on this 🙂 
I'd like to leave this open, since the core issue would still be publishing to hex.

Anyhow: thanks for the library! 👍

**Comment by kamilc** [comment]
Update: I tried to publish it to hex but it's unhappy about eiconv not being a hex package. So then I found out there is eiconv (finally) on hex but now it complains that it's in alpha... I'll have a look at it some time later in the day.

**Comment by maxbeizer** [comment]
@kamilc https://github.com/zotonic/eiconv/issues/4#issuecomment-387042308 . `eiconv` is now [`1.0.0`](https://hex.pm/packages/eiconv/1.0.0) !!

**Comment by skosch** [comment]
Thanks @maxbeizer for being on top of this! Would you like to amend your PR with the updated version number in the mix file before I merge it?

**Comment by maxbeizer** [comment]
closed by #81 ?

**Comment by skosch** [comment]
oops – yes, thanks.
